README.md

```diff
@@ -1 +1,38 @@
-# Capstone-project
\ No newline at end of file
+# Capstone-project
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+Capstone project
+
+## Description
+
+this is mi capstone project, the last project of the module 1 on microvers. i make a page about the book fear in Lima, this event is super interesting for the reading lovers.
+
+
+
+## GETTING STARTED
+
+copy this repo : https://github.com/jdbs9514/Capstone-project.git
+
+cd to the folder name.
+
+right click on the HTML file and open it in the browser.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Author
+
+Josué David Benavides Salinas
+
+- twitter:@JODA9514
+- Gmail:jdbs9514@gmail.com
+- GitHub: @jdbs9514
+
+## 📝 License
+
+This project is MIT.md licensed.
```

Comment:
_Kindly do these requirements to make a professional readme file._
- [ ] Please deploy your project and add a link in the readme file.
- [ ] Kindly add a design link in the acknowledgment section.
- [ ] Please add a creative common license in the license section.
 [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)
-  [ ] Kindly check your README file again. 
You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections, or typos. 
Also, you should change the readme template's sentence with your sentence.